### PR TITLE
[docs] modify introduction and add some detail

### PR DESCRIPTION
**Pull request by ferrirW**

### CODE REVIEW

**Review by humkum — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -20,21 +21,13 @@ It offers a variety of features:
...
-$ git clone git@github.com:apache/rocketmq-schema-registry.git
-$ cd rocketmq-schema-registry
-$ mvn clean package
-```
-
 #### Prepare storage layer
 
-Currently, only RocketMQ is supported as the storage layer. And relies on the Compact Topic feature of RocketMQ 5.0,
-although previous versions can also be worked, but there is a risk of data loss if the machine disk fails. Similarly,
+Currently, Schema registry only support RocketMQ storage, and relies on the Compact Topic feature of RocketMQ-5.0.
```
Comment (caigy) [inline README.md]: 'support' -> 'supports'

File: README.md [hunk 2]
```diff
@@ -20,21 +21,13 @@ It offers a variety of features:
...
+Previous rocketmq versions can also be worked, but there are some risk of data loss if the machine disk fails.
```
Comment (caigy) [inline README.md]: 'rocketmq' -> 'RocketMQ'

File: README.md [hunk 3]
```diff
@@ -51,24 +44,29 @@ $ nohup sh mqnamesrv &
...
-
 Take the build JAR in core/target/ and run `java -jar rocketmq-schema-registry-core-0.0.3-SNAPSHOT.jar` to start service.
 
 Then REST API can be accessed from http://localhost:8080/schema-registry/v1
 
 Swagger API documentation can be accessed from http://localhost:8080/swagger-ui/index.html
 
-Package management
---------------
+#### Customize configuration (Optional)
```
Comment (caigy) [inline README.md]: is it 'customized configurations'?

File: README.md [hunk 4]
```diff
@@ -119,6 +117,37 @@ $ curl -X GET http://localhost:8080/schema-registry/v1/cluster/{cluster-name}/te
...
+| FORWARD             | Add fields Delete optional fields | Last version          | Producers     |
+| FORWARD_TRANSITIVE  | Add fields Delete optional fields | All previous versions | Producers     |
+| FULL                | Modify optional fields            | Last version          | Any order     |
+| FULL_TRANSITIVE     | Modify optional fields            | All previous versions | Any order     |
+| NONE                | All changes are accepted          | Disabled              | Depends       |
+
+Architecture
+--------------
+
+Once Schema Registry enabled, the process for publish and subscribe will become to:
```
Comment (caigy) [inline README.md]: 'the process for publishing and subscribing' may be more appropriate.

**Review by caigy — CHANGES_REQUESTED** [review]

**Review by duhenglucky — APPROVED** [review]

**Pull request merged**